### PR TITLE
Enable file path arguments

### PR DESCRIPTION
**Pull request by joehandzik**
Signed-Off-By: Joe Handzik <jhandzik@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It adds new feature needed to simplify custom directory testing for the RN50 pipeline

#### What happened in this PR?
 - What solution was applied:
     *[Added flags to remove pipeline file paths and add custom paths ]*
 - Affected modules and functionalities:
     *[ test_RN50_data_pipeline additions only ]*
 - Key points relevant for the review:
     *[ Describe here what is the most important part that reviewers should focus on. ]*
 - Validation and testing:
     *[ Well tested against FileReadPipeline and TFRecordPipeline ]*
 - Documentation (including examples):
     *[NA]*

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by joehandzik** [comment]
Issue with my code or the build pipeline?

**Comment by JanuszL** [comment]
@joehandzik - CI. I will restart it. I will ping you if there is an issue. From my point of view it looks good.

File: dali/test/python/test_RN50_data_pipeline.py [hunk 1]
```diff
@@ -210,6 +211,18 @@ def define_graph(self):
                     help='Shuffle data when reading')
 parser.add_argument('--simulate_N_gpus', default=None, type=int, metavar='N',
                     help='Used to simulate small shard as it would be in a multi gpu setup with this number of gpus. If provided, each gpu will see a shard size as if we were in a multi gpu setup with this number of gpus')
+parser.add_argument('--remove_default_pipeline_paths', action='store_true',
+                    help="For all data pipeline types, remove the default values")
+parser.add_argument('--file_read_pipeline_paths', default=None, type=str, metavar='N',
+                    help='Override default FileReadPipeline paths. Separate multiple paths by commas')
```
Comment (klecki) [inline dali/test/python/test_RN50_data_pipeline.py:217]: Nitpick: it doesn't override the paths but adds additional ones.

Comment (klecki) [reply dali/test/python/test_RN50_data_pipeline.py:217]: I'm fine with either clearing the dictionary entry or rewording this.

Comment (joehandzik) [reply dali/test/python/test_RN50_data_pipeline.py]: Updated to "Add custom". Hope that covers it well enough!

**Comment by joehandzik** [comment]
Looks like DCO is flagging a conflict between my GitHub account email and NVIDIA email. Ideally I'd like to leave my signoff with "@nvidia.com", but let me know if that won't be possible.

**Comment by klecki** [comment]
!build

**Comment by JanuszL** [comment]
> Looks like DCO is flagging a conflict between my GitHub account email and NVIDIA email. Ideally I'd like to leave my signoff with "@nvidia.com", but let me know if that won't be possible.

You are part of the N family. This DCO is not a must in your case.

**Pull request merged**